app/src/main/java/de/tum/in/tumcampusapp/component/ui/cafeteria/interactors/FetchBestMatchMensaInteractor.kt

```diff
@@ -0,0 +1,14 @@
+package de.tum.`in`.tumcampusapp.component.ui.cafeteria.interactors
+
+import de.tum.`in`.tumcampusapp.component.ui.cafeteria.controller.CafeteriaManager
+import javax.inject.Inject
+
+class FetchBestMatchMensaInteractor @Inject constructor(
+        private val cafeteriaManager: CafeteriaManager
+) : Interactor<Int> {
```

Comment:
This class does surprisingly little work.
It's almost funny that we don't just use the called function directly :man_shrugging: